### DISCUSSION TITLE
How do it prevent calendar to popup when it get focus?

### DISCUSSION THREAD

**Discussion by anchan42** [primefaces/community#868] [PrimeVue] [answered]
I want the calendar to show only when user click the calendar button.
The default behavior is that the calendar pop up when the input field get focus.
How do I do that? 
I tried @focus.prevent but it didn't work.

**Comment by tugcekucukoglu** [comment, marked as answer]
You could set `showOnFocus` as false. The default value of this property is true.

**Reply by anchan42** [reply]
Thank you :)
